### ISSUE TITLE
Propose two new members to join to the TSC

### ISSUE THREAD

**Issue by Relequestual** [json-schema-org/TSC#7]
## Reason for vote

I would like to propose adding two members to the TSC.
These individuals are well known to the JSON Schema community and are active participants.

Matthew Adams (@mwadams) and Juan Viotti (@jviotti).

I call for two votes from the TSC as per our governance process of [decision making via vote](https://github.com/json-schema-org/community/blob/main/GOVERNANCE.md#decision-making-via-vote), for which I will create two comments in this Issue, linked below.

[Vote for Matthew Adams](https://github.com/json-schema-org/TSC/issues/7#issuecomment-2112646525)
[Vote for Juan Viotti](https://github.com/json-schema-org/TSC/issues/7#issuecomment-2112649649)

### Matthew Adams

[Matthew Adams](https://www.linkedin.com/in/matthewadamscto) is the primary maintainer of [Corvus.Net JSON Schema](https://github.com/corvus-dotnet/Corvus.JsonSchema) and Co-Founder of Endjin. Endjin is [committed to open source](https://endjin.com/what-we-do/open-source/) and Matthew works on Corvus.JsonSchema and several other projects as part of his work. Endjin is also a .NET Foundation Coporate Sponsor.

Recently, Matthew is [collaborating with JSON Schema](https://summerofcode.withgoogle.com/programs/2024/projects/JBmyddo1) in our participation of Google Summer of Code, by co-mentoring with @gregsdennis. (Although, this is happening through the Postman GSoC org). Very recently, Matthew published a [series of articles on JSON Schema design patterns](https://endjin.com/blog/2024/05/json-schema-patterns-dotnet-data-object)!

In chatting with Matthew on multiple locations, it's clear to me he has vast experience in programming, architecting, leadership, strategy, mediation, and processes. In addition, marketing and product/company positioning. I see matthew as someone who is wise and has a lot to offer.

### Juan Viotti

[Juan Viotti](https://www.linkedin.com/in/jviotti/) is the individual behind the groundbreaking award-winning research [JSON BinPack](https://jsonbinpack.sourcemeta.com) at the University of Oxford. This research [beat 40 years of benchmark records](https://arxiv.org/abs/2211.12799) in data serialization (partly by using JSON Schema, of course!).

Juan recently ended his contract at Postman (April 2024) after over two years working on major upgrades to their desktop platform, and is now Co-Founder of [Intelligence.AI](https://intelligence.ai/), a consultency company formed from the basis of a book he recently co-authored through O'Reily, which had a focus on JSON Schema. Juan also founded [Sourcemeta](https://www.sourcemeta.com), an open-source organization to house JSON BinPack and other JSON related creations, before joining Postman.

For a while Juan has been working on a C++ implementation of JSON Schema, and we look forward to seeing it completed and published. Recently, Juan created a [JSON Schema learning resource website](https://www.learnjsonschema.com), a [JSON Schema Awesome List](https://github.com/sourcemeta/awesome-jsonschema), and version converter [AlterSchema](https://github.com/sourcemeta/alterschema).

Juan has been a great sounding board on many occasions, and often provides us with insight and understanding into JSON Schema related research, and provides great technical feedback on specification work. Juan is also [collaborating with JSON Schema in Google Summer of Code](https://summerofcode.withgoogle.com/programs/2024/projects/OotP3B8B) via Postman to further the AlterSchema project.

## In Summary

Please take this opportunity to ask questions and do your own research.

Juan and Matthew are both invested in JSON Schema and Open Source in general, have shown they have both top-tier experience and expert knowledge in JSON Scheme, are very familiar with working in Open Source spaces, and have already provide much value to JSON Schema as a project.

I would be delighted to have them both join our TSC.

## Note

Our policy is that votes will close after 7 days and we must reach a 75% quorum for the vote to carry. I notice that at least one member of the TSC appears to be on vacation, so we will ideally wait for them to have the opportunity to ask questins and provide feedback/thoughts, and hopefully vote.

**Comment by Relequestual** [comment]
# Vote for Matthew Adams

Please signal your vote for Matthew Adams to be invited to join the TSC.

👍 In Favour
👎 Against
😕 Request 7 day extension

**Comment by Relequestual** [comment]
With FIVE votes in favour for both Juan and Matthew, I'd like to welcome you both to the TSC!

Matthew Adams (@mwadams) and Juan Viotti (@jviotti)!
